onnxruntime/core/framework/session_options.h

```diff
@@ -93,6 +93,7 @@ struct SessionOptions {
   // The configuration keys and value formats are defined in
   // /include/onnxruntime/core/session/onnxruntime_session_options_config_keys.h
   std::unordered_map<std::string, std::string> session_configurations;
+  std::unordered_map<std::string, const OrtValue*> initializers_to_share_map;
 
```

Comment:
Should this store an actual OrtValue instance instead of a pointer (given internally it has a shared_ptr to the allocator) so we manage the lifetime?

Reply:
I don't think it matters. The underlying buffer is still owned by the user. We do make a copy of OrtValue eventually when we add it to session state using SessionState::AddInitializedTensor.